### PR TITLE
AS-720: Fix display for gs uris that have a slash at the end

### PR DESCRIPTION
**Pull request by ansingh7115**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

**Comment by zarsky-broad** [comment]
This is a significant change in behavior, previously we just showed the file name because the path before it could be very long; this will fix the issue if an address is for a location and not a specific file, but are we ok with the change to what's displayed for files?

**Comment by ansingh7115** [comment]
That's true, I don't want to change the intent of this code - I updated it to continue showing only the file name while removing trailing slashes

File: src/components/UriViewer.js [hunk 1]
```diff
@@ -268,7 +268,7 @@ export const UriViewerLink = ({ uri, googleProject }) => {
         e.preventDefault()
         setModalOpen(true)
       }
-    }, [isGs(uri) ? _.last(uri.split('/')) : uri]),
+    }, [isGs(uri) ? _.last(uri.replace(/\/$/, '').split('/')) : uri]),
```
Comment (zarsky-broad) [inline src/components/UriViewer.js]:
```suggestion
    }, [isGs(uri) ? _.last(uri.split(/\/\b/)) : uri]),
```

Comment (zarsky-broad) [reply src/components/UriViewer.js]: What about this?

Comment (zarsky-broad) [reply src/components/UriViewer.js]: As a bonus, this will retain a trailing slash.

Comment (ansingh7115) [reply src/components/UriViewer.js]: That seems to work - it just leaves the extra slash at the end (which I think is probs fine)

Comment (ansingh7115) [reply src/components/UriViewer.js]: Updated

**Review by zarsky-broad — APPROVED** [review]

**Review by MatthewBemis — APPROVED** [review]
LGTM. Did not test myself!

**Pull request merged**